homeassistant/components/broadlink/entity.py

```diff
@@ -0,0 +1,36 @@
+"""Broadlink entities."""
+
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN
+
+
+class BroadlinkEntity(Entity):
```

Comment:
```suggestion
class BroadlinkEntity():
```
Is it necessary to inherit from `Entity` here?

Reply:
Its not strictly needed. I removed it